src/Introspector.php

```diff
@@ -21,11 +21,12 @@ public function introspect(): void
     {
         $client = $this->endpointConfig->makeClient();
 
-        $introspectionResult = $client->request(
-            Introspection::getIntrospectionQuery([
-                'directiveIsRepeatable' => true,
-            ])
-        );
+        $introspectionResult = $this->fetchIntrospectionResult($client, true);
```

Comment:
This will throw InvalidResponseException, see https://github.com/spawnia/sailor/blob/44678a8db4dd864c98c6bb36c3af999465cc8bf2/src/Response.php#L41

Reply:
The server I'm querying returns e400 instead of 200 on this kind of error

Reply:
But that's maybe invalid gql implementation of the server? Though there's no http spec yet.

I've tried another one and it works with it.

Reply:
Thanks for trying it out, now I will catch that too.